administration/src/util/translations/de.json

```diff
@@ -128,14 +118,19 @@
     "blueCardRequirementsTitle": "Ich erfülle folgende Voraussetzung für die Beantragung einer blauen Ehrenamtskarte:",
     "applicationStepPersonalData": "Persönliche Angaben",
     "applicationStepRequirements": "Voraussetzungen",
-    "applicationStepSubmit": "Antrag Senden"
+    "applicationStepSubmit": "Antrag Senden",
+    "title": "Bayerische Ehrenamtskarte beantragen",
+    "sentSuccessfully": "Erfolgreich gesendet",
+    "sentError": "Fehler beim Senden",
+    "nextStepButton": "Nächster Schritt",
+    "backButton": "Zurück",
+    "submitApplicationButton": "Antrag Abschicken",
+    "discardInputsTitle": "Alle Eingaben verwerfen?",
+    "discardInputsContent": "Wollen Sie wirklich alle Eingaben unwiderruflich löschen?",
+    "submitSuccessText": "Ihr Antrag für die Ehrenamtskarte wurde erfolgreich übermittelt.<br/>Über den Fortschritt Ihres Antrags werden Sie per E-Mail informiert.<br/>Sie können das Fenster jetzt schließen.",
+    "invalidInputError": "Ungültige bzw. fehlende Eingaben entdeckt. Bitte prüfen Sie die rot markierten Felder."
   },
-  "applications": {
-    "allApplications": "Alle Anträge",
-    "accepted": "Akzeptiert",
-    "rejected": "Abgelehnt",
-    "withdrawed": "Zurückgezogen",
-    "open": "Offen",
+  "applicationCard": {
```

Comment:
:question: maybe "applicationCard" and "applicationsOverview" should be the same category?
"applicationsOverview" sounds good to me.
"applicationCard" sounds like a card created from an application.

Reply:
"application" contains only database fields that shouldn't be edited. I think it is fine to keep them separately.
(maybe better name could be used, but I don't have a better suggestion)

Reply:
yes we can merge these two namespaces